readme.md

```diff
@@ -100,6 +100,85 @@ alfy.output([{
 
 <img src="media/screenshot-output.png" width="694">
 
+#### matches(input, list, [item])
+
+Returns an `Array` of items in `list` that case-insensitively contains `input`.
+
+```js
+alfy.matches('Corn', ['foo', 'unicorn']);
+//=> ['unicorn']
+```
+
+##### input
+
+Type: `string`
+
+Text to match against the `list` items.
+
+##### list
+
+Type: `Array`
+
+List to be matched against.
+
+##### item
+
+Type: `string` `Function`
+
+By default it will match against the `list` items.
+
+Specify a string to match against an object property:
+
+```js
+const list = [{
+	title: 'foo'
+}, {
+	title: 'unicorn'
+}];
+
+alfy.matches('Unicorn', list, 'title');
+//=> [{name: 'unicorn'}]
+```
+
+Or [nested property](https://github.com/sindresorhus/dot-prop):
+
+```js
+const list = [{
+	name: {
+		first: 'John',
+		last: 'Doe'
+	}
+}, {
+	name: {
+		first: 'Sindre',
+		last: 'Sorhus'
+	}
+}];
+
+alfy.matches('sindre', list, 'name.first');
+//=> [{name: {first: 'Sindre', last: 'Sorhus'}}]
+```
+
+Specify a function to handle the matching yourself. The function receives the list item and input as arguments and is expected to return a boolean whether it matches:
+
+```js
+const list = [{
+	name: 'foo',
+	sub: 'bar'
+}, {
+	name: 'unicorn',
+	sub: 'rainbow'
+}];
```

Comment:
It's actually intentionally not. The list is external data and often does not match the Alfred naming, and that's when you use the matching, you later map it into the correct names before using `alfy.output()`.